native/ffi/def.proto

```diff
@@ -28,6 +28,17 @@ message Request {
     IsAccountFollower is_account_follower = 22;
     IsSpaceFollower is_space_follower = 23;
     IsPostSharedByAccount is_post_shared_by_account = 24;
+    UnfollowSpace unfollow_space = 25;
```

Comment:
Can you put `FollowSpace` and `UnfollowSpace` in order one by one?

Reply:
Yes, I'm planning to order everything at the end of the final release.